test/test_ambiguities.jl

```diff
@@ -30,6 +30,11 @@ include("preamble.jl")
         if broken
             @test_broken num_ambiguities_ == num_ambiguities
         else
+            if num_ambiguities_ != num_ambiguities
+                @show exclude
+                println(strout)
+                println(strerr)
+            end
```

Comment:
How is this related to the rest of the changes in here?

Reply:
Just debugging for the relevant tests if they change reporting